frontend-ui/src/views/TaskDetailView.vue

```diff
@@ -71,7 +71,10 @@
                   <tr>
                     <th class="text-left w-20">Recipe</th>
                     <td>
-                      <router-link :to="{ name: 'schedule-detail', params: { scheduleName: task.schedule_name } }">
+                      <span v-if="task.schedule_name === null || task.schedule_name === 'none'">
```

Comment:
Why can we have `none` here? Looks like backend is not sending this value (anymore). I would rather get rid of this "oddity" unless we are sure we need it. To be removed in `WorkersView.vue` as well obviously.